.circleci/config.yml

```diff
@@ -375,3 +426,12 @@ workflows:
       - build-linux
       - build-macos
       - assert-test-linux
+  release:
+    jobs:
+      - arch-release:
+          filters:
+            branches:
+              ignore: /.*/
```

Comment:
Just wondering, should we only do the release once the semver is in the `master` branch?

Semi-related: I would like to rename that to `release` branch soon.

Reply:
This doesn't care about branches only tags. If you make the tag against `master`, it will run on that tag. If we make releases on another branch and don't want to make arch-releases for those, we may need to tweak the tag regex.

Reply:
I think @aykevl has been creating the tag on `dev` for each release.

Reply:
Release tags are created on the master branch by creating a GitHub release (which will automatically create a git tag).

> Semi-related: I would like to rename that to `release` branch soon.

Somewhat off-topic indeed but I agree, `release` describes the branch much better (alternatively, it could be `stable` which I've also seen on a few repositories).

Reply:
That being the case, we can merge this, and then see what happens when we cut our next release.